src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -433,192 +395,163 @@ createBlockHistoryTable =
         \ endingtxid UNSIGNED BIGINT NOT NULL, \
         \ CONSTRAINT blockHashConstraint UNIQUE (blockheight));"
 
-createVersionHistoryTable  :: BlockHandler SQLiteEnv ()
-createVersionHistoryTable =
-    callDb "createVersionHistoryTable" $ \db ->
-    exec_ db
-        "CREATE TABLE IF NOT EXISTS VersionHistory (version UNSIGNED BIGINT NOT NULL,\
-        \ blockheight UNSIGNED BIGINT NOT NULL,\
-        \ txid UNSIGNED BIGINT NOT NULL,\
-        \ CONSTRAINT versionConstraint UNIQUE (version, blockheight));"
-
-createUserTablesTable  :: BlockHandler SQLiteEnv ()
-createUserTablesTable =
-    callDb "createUserTablesTable" $ \db ->
-    exec_ db
-        "CREATE TABLE IF NOT EXISTS UserTables (tablename TEXT NOT NULL\
-        \ , createBlockHeight UNSIGNED BIGINT NOT NULL\
-        \ , version UNSIGNED BIGINT NOT NULL\
-        \ , CONSTRAINT versionTableConstraint UNIQUE\
-        \ (version, createBlockHeight, tablename));"
-
-createUserTable :: TableName -> BlockVersion -> BlockHandler SQLiteEnv ()
-createUserTable name b = do
-    callDb "createUserTable" $ \db ->
-        exec_ db $
-            "CREATE TABLE IF NOT EXISTS "
-            <> Utf8 (toS (flip T.snoc ']' $ T.cons '[' $ (asString name)))
-            <> " (rowkey TEXT\
-               \, blockheight UNSIGNED BIGINT\
-               \, version UNSIGNED BIGINT\
-               \, txid UNSIGNED BIGINT\
-               \, rowdata BLOB \
-               \, UNIQUE(blockheight, rowkey, txid));"
-    versionedTablesInsert name b
-
-handleVersion :: BlockHeight -> ParentHash -> BlockHandler SQLiteEnv TxId
-handleVersion bRestore hsh = do
-    bCurrent <- getBCurrent
-    vCurrent <- getVCurrent
-    checkHistoryInvariant
-
-    case compare bRestore (bCurrent + 1) of
-        GT -> internalError "handleVersion: Block_Restore invariant violation!"
-        EQ -> newChildBlock bCurrent vCurrent
-        LT -> rewindBlock vCurrent
-
+createTableCreationTable :: BlockHandler SQLiteEnv ()
+createTableCreationTable =
```

Comment:
local style is 4-space indent, pls fix everywhere

Reply:
done

Reply:
You say "done" but I see a bunch of 2-space indents below :)

E.g. four spaces after do

```haskell
foo :: Bar -> IO Baz
foo bar = do
    blah
    bar $ do
        etc etc etc
```